<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>485</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>830</width>
    <height>485</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>830</width>
    <height>485</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>651</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditname">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOrdNum">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditID">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>220</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="allDataViewButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Вывести все</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRem">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>280</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить запись (по ID)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="numberOfSec">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Общая стоимость</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAdd">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>250</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить запись</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Название тех средства</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Производитель</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>220</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCost">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>140</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdityear">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>180</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>120</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Инвентарный номер</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>160</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Год выпуска</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NameAZ">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Фамилия(А-Я)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NameZA">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>430</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Фамилия(Я-А)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OrdNumAZ">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>400</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>N Договора(А-Я)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OrdNumZA">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>430</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>N Договора(Я-А)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Cost09">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>400</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Стоимость (0-9)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Cost09_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>430</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Стоимость (9-0)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Year09">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата (0-9)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Year90">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата (9-0)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Сортировка</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
